server/services/content-types.js

```diff
@@ -47,12 +47,14 @@ const shouldAttributeBeIncluded = (attribute, includedRelations) => {
 }
 
 const getSelectedRelations = ({ schema, relations }) => {
-  return relations.reduce((acc, relation) => {
-    acc[relation] = {
-      select: Object.keys(schema[relation]).map((key) => key)
-    }
-    return acc
-  }, {})
+  return relations
```

Comment:
```suggestion
  return relations
    .reduce((acc, relation) => {
      // add the condition in here to avoid double loop
      if (relation in schema) {
        acc[relation] = {
          select: Object.keys(schema[relation]).map((key) => key)
        }
      }
      return acc
    }, {})
```